src/models/Profile.js

```diff
@@ -0,0 +1,57 @@
+export default (sequelize, DataTypes) => {
+  const Profile = sequelize.define('profiles', {
+    userId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      references: {
+        model: 'users',
+        key: 'id',
+      },
+    },
+    firstName: {
+      type: DataTypes.STRING,
+      allowNull: true
+    },
+    lastName: {
+      type: DataTypes.STRING,
+      allowNull: true
+    },
+    phoneNumber: {
+      type: DataTypes.STRING,
+      allowNull: true
+    },
+    departmentId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      references: {
+        model: 'departments',
+        key: 'id',
+      },
+    },
+    role: {
+      type: DataTypes.ENUM,
+      defaultValue: 'Staff',
+      allowNull: true
+    },
+    jobDescription: {
+      type: DataTypes.STRING,
+      allowNull: true
+    },
+    imageUrl: {
+      type: DataTypes.STRING,
+      allowNull: true
+    },
+    resetToken: {
+      type: DataTypes.STRING,
+      allowNull: true
+    },
+    isVerified: {
+      type: DataTypes.BOOLEAN,
+      allowNull: true
+    },
+  });
+  Profile.associate = (models) => {
+    // any associations
+  };
```

Comment:
Please add the necessary associations. I believe the department and user models are available

Reply:
@eokwukwe  yes I am presently on it.